src/workerd/api/pyodide/pyodide.c++

```diff
@@ -581,4 +586,69 @@ kj::String getPythonBundleName(PythonSnapshotRelease::Reader pyodideRelease) {
   return kj::str(pyodideRelease.getPyodide(), "_", pyodideRelease.getPyodideRevision(), "_",
       pyodideRelease.getBackport());
 }
+
+namespace api::pyodide {
+
+// Returns a string containing the contents of the hashset, delimited by ", "
+kj::String hashsetToString(kj::HashSet<kj::String>& set) {
+  if (set.size() == 0) {
+    return kj::str("");
+  }
+
+  kj::Vector<kj::StringPtr> elems;
+  for (const auto& e: set) {
+    elems.add(e);
+  }
```

Comment:
Consider using KJ_STACK_ARRAY here to avoid the additional heap allocation when the set it small enough.

Reply:
This is only used for debugging so I don't think it's worth these small optimisations. Also this code is copied from our internal repo.